app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+  end
+
+  def create
+    @user = User.find(params[:user_id])
```

Comment:
Kindly follow the DRY principle, I suggest using the `new` method to initialize a `new` like and reference it in the create method
- [ ] Please refactor this piece of code